centos-release-rpms-spec/centos-release-openshift-origin311.spec

```diff
@@ -0,0 +1,68 @@
+%global origin_release_dot 3.11
+%global origin_release_nodot 311
+
+Summary:   Yum configuration for OpenShift Origin %{origin_release_dot} packages
+Name:      centos-release-openshift-origin%{origin_release_nodot}
+Version:   1
+Release:   1%{?dist}
```

Comment:
Your change log lists 1-2 for the nvr. Should you update that here?

Reply:
can do